From: Michael Tautschnig <mt@debian.org>
Subject: Fix alignment error on ARM

Upstream status: A revised version of this patch is included in binutils/HEAD
---
 include/coff/external.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/coff/external.h b/include/coff/external.h
index 19636da..9e97fd1 100644
--- a/include/coff/external.h
+++ b/include/coff/external.h
@@ -50,7 +50,7 @@ typedef struct external_aouthdr
     char entry[4];	/* entry pt.				*/
     char text_start[4];	/* base of text used for this file 	*/
     char data_start[4];	/* base of data used for this file 	*/
-  }
+  } __attribute__((packed))
 AOUTHDR;
 
 #define AOUTHDRSZ 28
@@ -150,7 +150,7 @@ struct external_syment
   char e_type[2];
   char e_sclass[1];
   char e_numaux[1];
-};
+} __attribute__((packed));
 
 #define	SYMENT	struct external_syment
 #define	SYMESZ	18	
@@ -242,7 +242,7 @@ union external_auxent
     char x_tvlen[2];	/* length of .tv */
     char x_tvran[2][2];	/* tv range */
   } x_tv;		/* info about .tv section (in auxent of symbol .tv)) */
-};
+} __attribute__((packed));
 
 #define	AUXENT	union external_auxent
 #define	AUXESZ	18
